install_scripts/templates/kubernetes/yml-generate.sh

```diff
@@ -930,8 +958,7 @@ $weave_passwd_env
             seLinuxOptions: {}
           serviceAccountName: weave-net
           tolerations:
-            - effect: NoSchedule
-              operator: Exists
+            - operator: Exists
```

Comment:
this is a weird one. i maybe should experiment with changing this

Reply:
weave pods weren't getting scheduled on masters?

Reply:
they were not